### PR TITLE
Task/test amazon notebook

### PR DESCRIPTION
**Pull request by AbSsEnT**
Prepared a notebook "amazon_review_classification_giskard_example.ipynb" with examples of use of a standard giskard functionality: model and dataset wrapping, model scan.
https://linear.app/giskard/issue/GSK-1006/amazon-review-notebook

### CODE REVIEW

File: amazon_review_classification_reference.ipynb
Comment (rabah-khalek) [inline amazon_review_classification_reference.ipynb:1]: Could you please remove [amazon_review_classification_reference.ipynb](https://github.com/Giskard-AI/giskard-examples/pull/17/files#diff-9e871edec5bfeda3564c19fac7fdb9e4832d0628e60b892a587caa26e6be5de6) and instead put a link towards the original notebook inside [amazon_review_classification_giskard_example.ipynb](https://github.com/Giskard-AI/giskard-examples/pull/17/files#diff-8d61555e07a9d37f09f0299d039f4894515bce2512086b3d64637c58597ccaef)?

File: amazon_review_classification_giskard_example.ipynb
Comment (rabah-khalek) [inline amazon_review_classification_giskard_example.ipynb:1]: Could you just name [amazon_review_classification_giskard_example.ipynb](https://github.com/Giskard-AI/giskard-examples/pull/17/files#diff-8d61555e07a9d37f09f0299d039f4894515bce2512086b3d64637c58597ccaef), `amazon_review_classification_sklearn.ipynb` instead?

Comment (rabah-khalek) [reply amazon_review_classification_giskard_example.ipynb:1]: Could you add `!pip install nltk giskard` in a separate cell at the beginning?

Comment (rabah-khalek) [reply amazon_review_classification_giskard_example.ipynb:1]: Could you also remove the last line from:
```py
reviews_df = download_data()
reviews_df = preprocess_data(reviews_df)
reviews_df
```

Comment (rabah-khalek) [reply amazon_review_classification_giskard_example.ipynb:1]: Add two additional parts to the notebooks:
1. https://giskard.readthedocs.io/en/latest/getting-started/scan/index.html#upload-your-model-and-dataset-to-giskard-ui
2. https://giskard.readthedocs.io/en/latest/getting-started/test-suite/index.html#create-execute-a-test-suite together with https://giskard.readthedocs.io/en/latest/getting-started/test-suite/index.html#save-a-test-suite

**Review by rabah-khalek — CHANGES_REQUESTED** [review]

File: amazon_review_classification_reference.ipynb
Comment (AbSsEnT) [reply amazon_review_classification_reference.ipynb:1]: - I'll remove it;
- The link to it is already provided in the header of notebook.

**Review by rabah-khalek — APPROVED** [review]

**Pull request merged**